samples/README.md

```diff
@@ -110,9 +110,20 @@ cases above.
 
 A comment on a given line provides information about the following line.
 
-Such a comment contains one of 3 special sequences:
+Such a comment contains one of 5 special sequences. The first 3 cover case 1
+from above:
 
--   `jspecify_nullness_paradox`: for case 1 above
+-   `jspecify_multiple_annotations`: for cases like `@Nullable
```

Comment:
This could also be "conflicting".  When Jspecify expands, it may permit multiple annotations at a single location, such as `int @NonNull @NonEmpty []`.

Reply:
Makes sense. Done.

(Mike and I had talked elsewhere about using the term "inconsistent." We set it aside because readers might also conclude that an annotation can be "inconsistent" with a corresponding annotation on a supermethod, for example. While we could have the same concern about "conflicting," I feel that it's clearer somehow. Perhaps it's more natural to assume that a "conflict" occurs between two things that are sitting right next to one another in the source file, whereas "inconsistency" is the kind of thing that you might see even between 2 completely separate codebases.)

I'll leave this PR open for a couple days to collect feedback from @kevin1e100 and other interested parties. After that, I'll make the changes and merge it. As always, we can revisit as needed.